docker-hub/release-notes.md

```diff
@@ -11,6 +11,37 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+
+## 2022-7-22
+### Enhancement
+
+You can now invite members to your organization with a CSV file containing their email addresses. The CSV file can either be a file you create for this specific purpose or one that’s extracted from another in-house system.
+
+## 2022-07-19
+
+### Enhancement
+
+SCIM is now available for organizations with a Docker Business subscription using an Azure Ad identity provider.
+
+## 2022-06-23
+
+### New feature
+
+With SCIM, you can manage users within your Okta identity provider (IdP). In addition, you can enable SCIM on organizations that are part of the Docker Business subscription.
```

Comment:
Can we also add a link to the SCIM topic from the release notes? Suggest we do the same for other features in this PR so users can find more information if required.

Reply:
Thanks Usha! Updated